index.js

```diff
@@ -0,0 +1,151 @@
+// task__1
+
+const sayHello = function (name) {
+    if (name == 'Mark') {
+        return console.log(`Hi, ${name}`);
+    } else {
+        return console.log(`Hello, ${name}`);
+    }
+};
+
+// const sayHello = name => {
+//     if (name == 'Mark') {
+//         return console.log(`Hi, ${name}`);
+//     } else {
+//         return console.log(`Hello, ${name}`);
+//     }
+// };
+
+sayHello('Mark');
+sayHello('Oleg');
+sayHello('Viktor');
+
+
+
+
+// task__2 
+
+const result = function(n, m) {
```

Comment:
weird naming. Rename you function in according with my explanations.
Tips: your function calculates hypotenuze. Choose the name in according with what your function actually do:)